### PR TITLE
Set a default `fwup_devpath` in configurator

### PR DESCRIPTION
**Pull request by ConnorRigby**

### CODE REVIEW

**Review by jjcarstens — APPROVED** [review]

File: lib/nerves_hub_link/configurator.ex [hunk 1]
```diff
@@ -13,11 +13,26 @@ defmodule NervesHubLink.Configurator do
               remote_iex: false,
               socket: [],
               ssl: []
+
+    @type t() :: %__MODULE__{
+            device_api_host: String.t(),
+            device_api_port: String.t(),
+            device_api_sni: charlist(),
+            fwup_public_keys: [binary()],
+            fwup_devpath: String.t(),
```
Comment (fhunleth) [inline lib/nerves_hub_link/configurator.ex:22]:
```suggestion
            fwup_devpath: Path.t(),
```

File: lib/nerves_hub_link/configurator.ex [hunk 2]
```diff
@@ -93,4 +109,9 @@ defmodule NervesHubLink.Configurator do
 
     %{config | fwup_public_keys: config.fwup_public_keys ++ fwup_public_keys}
   end
+
+  @spec maybe_add_fwup_devpath(Config.t()) :: Config.t()
+  defp maybe_add_fwup_devpath(config) do
```
Comment (fhunleth) [inline lib/nerves_hub_link/configurator.ex:114]: This doesn't feel like a "maybe" unless I'm missing something.

File: lib/nerves_hub_link/configurator.ex (hunk 1 above)
Comment (ConnorRigby) [reply lib/nerves_hub_link/configurator.ex:22]: Path.t() is `iodata` which i don't think would actually work

File: lib/nerves_hub_link/configurator.ex (hunk 2 above)
Comment (ConnorRigby) [reply lib/nerves_hub_link/configurator.ex:114]: Whoops, that was supposed to be `put_new` . just a sec

File: lib/nerves_hub_link/configurator.ex (hunk 1 above)
Comment (ConnorRigby) [reply lib/nerves_hub_link/configurator.ex:22]: i'm totally wrong. we just updated teh other lib to support this

**Review by jjcarstens — APPROVED** [review]

**Pull request merged**